pkg/agent/client/util.go

```diff
@@ -0,0 +1,130 @@
+package client
```

Comment:
It would be nice that we could use the `api.ProtoToRegistryEntry` and `api.ProtoToBundle` functions which should probably belong to a common package and not be under the server. I see that's due to the way that trust domains are handled. It would be nice to have a solution for this although it doesn't seem necessary to address that as part of this PR.